packages/replicache/src/persist/collect-idb-databases.ts

```diff
@@ -305,6 +305,7 @@ function canDatabaseBeCollectedAndGetDeletedClientIDs(
 ): Promise<
   [canCollect: false] | [canCollect: true, deletedClientIDs: ClientID[]]
 > {
+  // TODO: Also see if there is a deleted-clients head and include that information.
```

Comment:
When doing that I think it would make sense to have Replicache be responsible of this head instead of zero dealing with it.

Reply:
This does seem to fit more naturally in the Replicache layer.  It also would be a nice addition to Replicache (though we may never prioritize shipping it), as Replicache currently has a problem where the server can never correctly delete client records (https://www.notion.so/replicache/Deleting-Client-entries-from-backend-database-bdc4592792f94e7489507bba693c8359?pvs=4#bf968d1ca19b4f2e8d909d3350e353c5)